main.js

```diff
@@ -0,0 +1,147 @@
+const VALIDATE = (prop) => {
+    if (prop === undefined || prop === null) {
```

Comment:
It can be simplified
```suggestion
   return prop === undefined || prop === null;
```